samples/highcharts/annotations-advanced/controllable/demo.js

```diff
@@ -197,6 +197,93 @@ var chart = Highcharts.chart('container', {
                             Math.max(target.options.r - radiusTranslation, 5)
                         );
 
+                        target.redraw(false);
+                    }
+                }
+            }]
+        }, {
+            type: 'ellipse',
+            point: {
+                x: 10,
+                y: 6,
+                xAxis: 0,
+                yAxis: 0
+            },
+            angle: 45,
+            rx: 40,
+            ry: 30,
+            controlPoints: [{
```

Comment:
In the demo, I'm able to edit each annotation, however, when I try to change the ellipse it acts strange.